examples/Exploring-different-models.ipynb

```diff
@@ -0,0 +1,802 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    from example_utils import workflow_fit_transform
</pre><p>I pitched "demo" instead of <code>merlin.models.data</code> when I reviewed Benedikt's notebook.  If the team can settle on either <code>demo</code> or <code>example</code>, it'd be a nice consistency.  I favor <code>demo_utils</code> FWIW.    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/256/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>